ahnlich/types/src/ai/query.rs

```diff
@@ -1 +1,53 @@
+use super::{AIModel, AIStoreType};
+use crate::keyval::{StoreKey, StoreName, StoreValue};
+use crate::metadata::{MetadataKey, MetadataValue};
+use crate::predicate::PredicateCondition;
+use crate::similarity::Algorithm;
+use serde::{Deserialize, Serialize};
+use std::collections::HashSet;
+use std::num::NonZeroUsize;
 
+#[derive(Debug, Clone, Serialize, Deserialize, PartialEq, Eq)]
+pub enum AiQuery {
+    CreateStore {
+        r#type: AIStoreType,
```

Comment:
We already know it's just binary and raw string so AIStoreType should be a unit enum 

```rust
pub enum AIStoreType {
 RawString,
 Binary,
}
```

Reply:
Yeah, the emphasis was to give you an idea of how the query would look like